docs/specviz/import_data.rst

```diff
@@ -60,3 +65,27 @@ This method works well for data files that ``specutils`` understands.  However,
 
 For more information about using the Specutils package, please see the
 `Specutils documentation <https://specutils.readthedocs.io>`_.
+
+Loading multiple spectra via the API
+------------------------------------
+In addition to loading single spectra as above, in some cases it may be useful to load multiple related
+spectra at once into the Jdaviz application. The :meth:`jdaviz.configs.specviz.helper.Specviz.load_data` accepts
+a `~specutils.SpectrumList` object, in which case it will both load the individual `~specutils.Spectrum1D`
+objects in the list and additionally attempt to stitch together the spectra into a single data object so that
+they can be manipulated and analyzed in the application as a single entity::
```

Comment:
Similarly, we might want to add a few sentences on how the stitching works.  Or how Jdaviz attempts to combine the spectra?  Is the stitching automatic or does the user have control over it from a plugin, layer, or notebook?